modules/ROOT/pages/resolve-private-domains-vpc-task.adoc

```diff
@@ -28,9 +28,9 @@ image::internal-dns-configuration.png[]
 When resolving a domain name in your internal network, consider the following:
 
 * The IP address you configure is for the DNS server of your internal network. It is not the IP address of the service you are connecting to.
-* The maximum number of characters of all the domains you configure must not exceed 239 characters. For example, adding the domains `mydomain1.com` and `mydomain2.com` requires 28 characters. Each domain name consumes an additional character because in the DNS hierarchy all domains end with a "." character.
-* Avoid entering hostnames to ensure that you do not exceed the 239 character limit.
-* All the DNS servers entered must present valid records for all the domains listed to keep a healthy view over the DNS namespace.
+* The maximum number of characters of all the domains you configure must not exceed 229 characters. For example, adding the domains `mydomain1.com` and `mydomain2.com` requires 28 characters. Each domain name consumes an additional character because in the DNS hierarchy all domains end with a "." character.
+* Avoid entering hostnames to ensure that you do not exceed the 229 character limit.
+* All DNS servers must present the same view of the DNS namespace. This means each server has valid and identical records for all of the private domains listed.
```

Comment:
Is this standard language: *present the same view* of the DNS namespace?

Reply:
How would you phrase it instead?

Reply:
Each DNS server must include valid and identical records for all of the private domains listed.

Is that better?